### PR TITLE
Ca 170 answer surveys

### PR DESCRIPTION
**Pull request by DawsonDease**

### CODE REVIEW

File: components/Survey.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React, { useContext } from "react";
+import { Title, Text, Button } from "react-native-paper";
+import PropTypes from "prop-types";
```
Comment (JaycobDuffel) [inline components/Survey.js:3]: Prop types aren't implemented

File: screens/SurveyScreen.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import Survey from "../components/Survey"
+import React, { useContext } from "react";
+import { Title, Text, Button } from "react-native-paper";
+import PropTypes from "prop-types";
+import ContentCard from "./ContentCard";
+import AuthContext from "../auth/Context";
+import { trackClick } from "./TrackClick";
+import Survey from "../components/Survey"
+
```
Comment (JaycobDuffel) [inline screens/SurveyScreen.js:4]: remove unused imports

**Review by JaycobDuffel — CHANGES_REQUESTED** [review]

**Review by katiemcgoff — APPROVED** [review]

File: config/config.js [hunk 3]
```diff
@@ -2,12 +2,15 @@ export const apiConfig = {
   baseUrl: "http://127.0.0.1:8000/",
   // baseUrl: "http://10.0.2.2:8000/",
   // baseUrl: "http://192.168.1.71:8000/",
+  //baseUrl: "http://192.168.0.35:8000/",
+
```
Comment (katiemcgoff) [inline config/config.js:5]: This commented-out line shouldn't be checked in.

File: screens/NotificationScreen.js [hunk 4]
```diff
@@ -6,6 +6,8 @@ import AuthContext from "../auth/Context";
 import { apiConfig } from "../config/config";
 import FixedText from "../components/FixedText";
 import Question from "../components/Question";
+import Survey from "../components/Survey"
+import { HOST_WITH_PORT } from "../environment";
```
Comment (katiemcgoff) [inline screens/NotificationScreen.js]: HOST_WITH_PORT isn't used.

File: screens/QuestionScreen.js [hunk 5]
```diff
@@ -1,24 +1,33 @@
...
-import { Appbar } from "react-native-paper";
-import { SafeAreaView, Text } from "react-native";
+import { SafeAreaView} from "react-native";
 import { apiConfig } from "../config/config";
 import axios from "axios";
 import Question from "../components/Question";
 import AuthContext from "../auth/Context";
 import { trackClick } from "../components/TrackClick";
+import {Text} from "react-native-paper";
+import { HOST_WITH_PORT } from "../environment";
```
Comment (katiemcgoff) [inline screens/QuestionScreen.js]: HOST_WITH_PORT isn't used.

**Review by katiemcgoff — CHANGES_REQUESTED** [review]
Unused imports (plus what Jaycob found), otherwise good.

**Review by JaycobDuffel — APPROVED** [review]

**Review by katiemcgoff — APPROVED** [review]

**Pull request merged**